packages/css-library/README.md

```diff
@@ -14,4 +14,4 @@ To generate stylesheets, run `yarn build:stylesheets` or `yarn build:minify`
 
 The tokens generated by this library will be available in several different formats so that various tools and environments can use them.
 
-To generate CSS token files, run `yarn build:tokens`
\ No newline at end of file
+To generate CSS token files or a Figma token file, run `yarn build:tokens`
```

Comment:
We will be adding a few more token types (react-native, sketch) so instead of us updating the readme for each of those, maybe we should just take out the "CSS" part from the sentence:

> To generate all token files, run `yarn build:tokens`